ads/alp/OWNERS.yaml

```diff
@@ -0,0 +1,4 @@
+# For an explanation of the OWNERS.yaml rules and syntax, see:
+# https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
+
+- ampproject/wg-ads
```

Comment:
Since this group is already in the OWNERS file for the `ads` directory, it automatically inherits ownership of all subdirectories. So, this file and  `ads/inabox/OWNERS.yaml` are not required

Reply:
removed